src/utils/data.js

```diff
@@ -52,3 +52,15 @@ export async function getData() {
     }
   });
 }
+
+export async function getSearchData(str) {
+  try {
+    console.log("getting search data");
+    let res = await fetch(
+      `https://api.mapbox.com/geocoding/v5/mapbox.places/${str}.json?types=country,region,postcode,district,place&access_token=pk.eyJ1IjoibWFwcGluZ2FjdGlvbiIsImEiOiJjazZrMTQ4bW4wMXpxM251cnllYnR6NjMzIn0.9KhQIoSfLvYrGCl3Hf_9Bw`
```

Comment:
Probably good to pull out the API token and export it from `src/utils/config.js`

Reply:
put the api token in config